### PR TITLE
discord-canary: 0.0.98 -> 0.0.99

### PR DESCRIPTION
**Pull request by tadeokondrak**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by kraem** [comment]
This seems to be outdated already, `0.0.102` seems to be the latest. Make the bump and I'll test it out :)

File: pkgs/applications/networking/instant-messengers/discord/default.nix [hunk 1]
```diff
@@ -27,10 +27,10 @@ in {
     pname = "discord-canary";
     binaryName = "DiscordCanary";
     desktopName = "Discord Canary";
-    version = "0.0.98";
+    version = "0.0.99";
```
Comment (kraem) [inline pkgs/applications/networking/instant-messengers/discord/default.nix]: `version = "0.0.102";`

File: pkgs/applications/networking/instant-messengers/discord/default.nix [hunk 2]
```diff
@@ -27,10 +27,10 @@ in {
...
     src = fetchurl {
       url = "https://dl-canary.discordapp.net/apps/linux/${version}/discord-canary-${version}.tar.gz";
-      sha256 = "0raqsfakjbcsh7g280yi7sg2jsrmy2q4jldg73wb868z35radld4";
+      sha256 = "1jld8qimz85g3655210shpyjvmi3rx4lfd6bsj9v8sxxmywyp9ih";
```
Comment (kraem) [inline pkgs/applications/networking/instant-messengers/discord/default.nix]: `sha256 = "1zl53vprkwbkfn3v0lddpp1swignqjxksi4qxl5zqj7482j372a4";`

**Review by kraem — CHANGES_REQUESTED** [review]
This seems to compile and run fine for me :)

**Comment by tadeokondrak** [comment]
#81050

**Pull request closed without merging**